stock_owner_restriction/models/stock_picking_type.py

```diff
@@ -9,9 +9,9 @@ class StockPickingType(models.Model):
 
     owner_restriction = fields.Selection(
         [
-            ("standard_behavior", "Standard behavior"),
-            ("unassigned_owner", "Unassigned owner"),
-            ("picking_partner", "Picking partner"),
+            ("standard_behavior", "Standard Behavior"),
```

Comment:
Please don't change this that forces to retranslate, and it's not more correct putting both in upper.